arrabiata/src/curve.rs

```diff
@@ -0,0 +1,73 @@
+//! This file defines a trait similar to [kimchi::curve::KimchiCurve] for Pallas and
+//! Vesta. It aims to define all the parameters that are needed by a curve to be
+//! used in Arrabiata. For instance, the sponge parameters, the endomorphism
+//! coefficients, etc.
+//! The goal of this trait is to parametrize the whole library with the
+//! different curves.
+
+use ark_ec::short_weierstrass::Affine;
+use kimchi::curve::{pallas_endos, vesta_endos};
+use mina_curves::pasta::curves::{pallas::PallasParameters, vesta::VestaParameters};
+use mina_poseidon::poseidon::ArithmeticSpongeParams;
+use poly_commitment::commitment::{CommitmentCurve, EndoCurve};
+
+/// Represents additional information that a curve needs in order to be used
+/// with Arrabiata.
+pub trait ArrabiataCurve: CommitmentCurve + EndoCurve {
```

Comment:
Is it necessarry to introduce another trait ?
I don't see anything that isn't in KimchiCurve
(Which is not a surprise, as both Kimchi and Arrabiata use a cycle of curve for plonk verification operations)